src/creatures/players/player.cpp

```diff
@@ -9630,6 +10399,52 @@ void Player::sendLootContainers() const {
 	}
 }
 
+// OTCR Features
+
+void Player::sendAttachedEffect(const std::shared_ptr<Creature> &creature, uint16_t effectId) const {
+	if (!client || !creature) {
+		return;
+	}
+
+	if (creature->getPlayer()) {
```

Comment:
If this is only executed for player, it is better to change this to stay in the player class and change the default to "Player" instead of creature (same for the others)